daemon/src/maker.rs

```diff
@@ -23,19 +26,54 @@ mod wire;
 #[database("maker")]
 pub struct Db(sqlx::SqlitePool);
 
+#[derive(Clap)]
+struct Opts {
+    /// The port to listen on for p2p connections.
```

Comment:
wow, I've just realised that Clap parses these comments into the CLI interface. super cool! 😎